src/subscriber.rs

```diff
@@ -207,6 +208,11 @@ impl<T: ?Sized> InactiveSubscriber<T> {
     pub fn subscription_state(&self) -> SubscribeState {
         self.ffi_sub.subscription_state()
     }
+
+    /// Releases a raw sample which will not be used anymore
+    pub fn release_raw(&self, sample: RawSample<T>) {
```

Comment:
What happens when I call `release_raw` multiple times with the same cloned sample? I think `RawSample` can be cloned and copied.

I think it should be an `unsafe` function and the user has to ensure that:
* the sample was not released before
* the sample was taken/acquired via the subscriber it is being released

Reply:
Currently nothing bad happens since the C++ code base takes care of this. iceoryx just prints a warning with `[Warning]: ICEORYX error! POPO__CHUNK_SENDER_INVALID_CHUNK_TO_FREE_FROM_USER`. Since nothing bad happens I thought it does not need to be `unsafe`

Reply:
Ouh, I thought that iceoryx would handle it in a more severe manner. In this particular case, I agree with you but it still feels weird and a little bit wrong.

Reply:
Indeed. There is quite a lot which can co wrong but all is handled in a graceful way. I don't know anymore why we chose this route at that time